src/examples/demo_face_sequential.cpp

```diff
@@ -43,18 +43,20 @@ FaceExtractor* face_extractor =
     new FaceExtractor("models/face_extraction/iresnet18_1.param",
                       "models/face_extraction/iresnet18_1.bin", 112, true);
 
-FaceManager* face_manager = new FaceManager("./data", 1000, 512, 1, 1.01);
+FaceManager* face_manager = new FaceManager("data.hnsw", 1000, 1, 512, 1.01);
 
 int main(int argc, char** argv) {
   // Read faces and name to register
-  if (argc < 3) {
+  if (argc < 2) {
     std::cout << "Please provide name and an image to register" << std::endl;
     std::cout << "Example: ./demo_face_sequential data/my_face.jpg "
```

Comment:
@haok61bkhn Remove the space in the end "Example: ./demo_face_sequential data/my_face.jpg "